doc/contributing/collaborator-guide.md

```diff
@@ -378,6 +380,20 @@ an effort to determine the potential impact of the change in the ecosystem. Use
 If a change will cause ecosystem breakage, then it is semver-major. Consider
 providing a Public API in such cases.
 
+### Breaking changes to console APIs
+
+Console APIs are a debugging tool, and the format of their output should generally not
+considered stable.
+However due to the nature of the Node.js ecosystem, users rely on stability of
+the output of some of these APIs for snapshot testing, parsers, etc...
+To avoid breaking changes, some behaviors should be considered stable.
+The following changes in the output are to be considered breaking changes:
+
+* `console.table` (padding, alignment, etc...).
+* changing the color of all console methods.
+* changing the signature of all console methods.
```

Comment:
The signature is as-stable as every other API call and the rest aren't stable (are they?)

Reply:
I dont know if the signature is considered stable but not the output, if its obvious Im gonna remove it